package.json

```diff
@@ -28,7 +28,9 @@
     "html-encoding-sniffer": "^1.0.1",
     "nwmatcher": ">= 1.3.9 < 2.0.0",
     "parse5": "^1.5.1",
+    "pn": "^1.0.0",
     "request": "^2.79.0",
+    "request-promise-native": "^1.0.3",
```

Comment:
Why a `request` variant?  I thought [`jsdom` had perf problems with it](https://github.com/tmpvar/jsdom/pull/1630).